src/styles/utils/_reset.scss

```
@@ -0,0 +1,27 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+h1,
+h2,
+h3 {
+  margin: 0;
+  font-weight: normal;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

```suggestion
body,
p {
  margin: 0;
}
```